### PR TITLE
Minor edits to miscellaneous tuts

### PR DESCRIPTION
**Pull request by rochisha0**
The last PR #79 had some mistakes that this pull request aims to fix. These issues were pointed out by @hodgestar 

1. The title of Single photon interference tutorials was not rendered correctly.
2. The index of tutorial page had an error

### CODE REVIEW

File: tutorials-v5/miscellaneous/single-photon-interference.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ jupyter:
     name: python3
 ---
 
-## QuTiP Lecture: Single-photon Interference
+# Single-photon Interference
```
Comment (hodgestar) [inline tutorials-v5/miscellaneous/single-photon-interference.md:15]:
```suggestion
# Lecture: Single-photon Interference
```

File: tutorials-v4/miscellaneous/single-photon-interference.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ jupyter:
     name: python3
 ---
 
-## QuTiP Lecture: Single-photon Interference
+# QuTiP Lecture: Single-photon Interference
```
Comment (hodgestar) [inline tutorials-v4/miscellaneous/single-photon-interference.md:15]:
```suggestion
# Lecture: Single-photon Interference
```

**Review by hodgestar — COMMENTED** [review]
The name of the notebook should be consistent between v4 and v5. I agree the "QuTiP Lecture" was a bit redundant but I think perhaps "Lecture:" makes it clearer what kind of material the notebook contains (i.e. it's a presentation on single photon interference).

**Comment by rochisha0** [comment]
@hodgestar Thank you for the comments, I have made the necessary changes.

**Review by hodgestar — APPROVED** [review]

**Review by hodgestar — APPROVED** [review]

**Pull request merged**